telemetry_api/lib/telemetry_api/operators.ex

```diff
@@ -98,46 +98,23 @@ defmodule TelemetryApi.Operators do
 
   ## Examples
 
-      iex> update_operator_version(%{field: value})
+      iex> update_operator(%{field: value})
       {:ok, %Ecto.Changeset{}}
 
-      iex> update_operator_version(%{field: bad_value})
+      iex> update_operator(%{field: bad_value})
       {:error, string}
 
   """
-  def update_operator_version(%{"version" => version, "signature" => signature}) do
-    with {:ok, address} <- SignatureVerifier.recover_address(version, signature) do
+  def update_operator(attrs) do
+    with {:ok, address} <- SignatureVerifier.recover_address(attrs["version"], attrs["signature"]) do
```

Comment:
I guess we should keep the old version of this. As suggested [here](https://github.com/yetanotherco/aligned_layer/pull/1000#discussion_r1784944450).

Reply:
I made some changes, have a look and let me know if you think I should change something else!